transform-plugins/docs/ValueMapper-transform.md

```diff
@@ -0,0 +1,82 @@
+# ValueMapper Transform
+
+
+Description
+-----------
+Value Mapper is a transform plugin that maps string values of a field in the input record to mapping value using mapping dataset.
+Mappings are usually stored in KeyValue dataset. Value Mapper provides you a simple alternative for manipulating the input data by using mapping dataset
+
+
+Use Case
+--------
+Suppose, If you want to replace language codes in the input record field to language description.
```

Comment:
One use is to replace language codes in the input record field with an appropriate language description:
     **Source field name:** language_code
     **Target field name:** language_desc
     **Mappings, source to target:** DE/German, ES/Spanish, EN/English